game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```
@@ -205,7 +204,7 @@ public void mouseMoved(final MouseEvent e) {
       });
     });
     addScrollListener((x2, y2) -> SwingUtilities.invokeLater(this::repaint));
-    executor.execute(() -> recreateTiles(data, uiContext));
+    executor.submit(() -> recreateTiles(data, uiContext));
```

This should stay execute, otherwise any errors might get swallowed